src/CloudFlare/IpRewrite.php

```diff
@@ -2,65 +2,72 @@
 
 namespace CloudFlare;
 
-class IpRewrite {
-    static protected $is_cf = NULL;
-    static protected $original_ip = FALSE;
-    static protected $rewritten_ip = FALSE;
-    
-    static protected $cf_ipv4 = array(
-        "199.27.128.0/21",
-        "173.245.48.0/20",
-        "103.21.244.0/22",
-        "103.22.200.0/22",
-        "103.31.4.0/22",
-        "141.101.64.0/18",
-        "108.162.192.0/18",
-        "190.93.240.0/20",
-        "188.114.96.0/20",
-        "197.234.240.0/22",
-        "198.41.128.0/17",
-        "162.158.0.0/15",
-        "104.16.0.0/12",
-        "172.64.0.0/13"
+require_once 'vendor/autoload.php';
+
+class IpRewrite
+{
+    protected static $is_cf = null;
+    protected static $original_ip = false;
+    protected static $rewritten_ip = false;
+
+    protected static $cf_ipv4 = array(
```

Comment:
Lets add a comment saying these IP ranges are found at https://www.cloudflare.com/ips/ in case anyone needs to update them in the future.